libs/community/langchain_community/vectorstores/duckdb.py

```diff
@@ -76,10 +82,10 @@ def __init__(
         *,
         connection: Optional[Any] = None,
         embedding: Embeddings,
-        vector_key: str = "embedding",
-        id_key: str = "id",
-        text_key: str = "text",
-        table_name: str = "vectorstore",
+        vector_key: str = DEFAULT_VECTOR_KEY,
+        id_key: str = DEFAULT_ID_KEY,
+        text_key: str = DEFAULT_TEXT_KEY,
+        table_name: str = DEFAULT_TABLE_NAME,
```

Comment:
this is a change from before - used to be `vectorstore`, now is `embeddings`, is that intentional?

Reply:
Well, yes.
They were not unified and it caused issues - you created the table with one name and tried to search in the table using the second name.